### ISSUE TITLE
Our options are under ".NET Core" catagory

### ISSUE THREAD

**Issue by davkean** [dotnet/project-system#2393]
Our project system is supported for projects outside of .NET Core, why are we showing up in .NET Core catagory?

![image](https://cloud.githubusercontent.com/assets/1103906/26818731/3467eb38-4adf-11e7-8154-b719e94c5188.png)

**Comment by davkean** [comment]
This also feels like we're shipping the org chart....

**Comment by srivatsn** [comment]
It's definitely not the org chart because the same org owns both project systems :) 

Any suggestions for names? To customers, currently, the option in this page primarily affect .NET Core and .NET Standard projects - yes folks can create projects targeting desktop using the SDK but there are no templates for that and so if you already know how to do that, then you are unlikely to be confused by this. That's not to say that this is a great name but we couldn't think of a better name either - the technically correct names would be "Projects that have a TargetFramework property" or "SDK-style projects" or some such which doesn't convey anything to someone who doesn't already have context.

**Comment by jmarolf** [comment]
Maybe just call it "advanced"? Most people likely won't know what this option will do.

**Comment by davkean** [comment]
We should have put this on _Build and Run_ page.

**Comment by panopticoncentral** [comment]
No, please, not "Build and Run." It's already bad enough that several of the options on that page apparently *only* apply to C++, even though there is no actual visible way to know that. I always wondered how "MSBuild project build log file verbosity" applied to C# and, hey, guess what? It doesn't!

I think we should have a separate page. I was at a loss for what the name should be. I suppose we could do like C++ and have it under "C#/VB/F# Project Settings." Is that too wordy? And are we sure we'll never add another project type?

The other direction is to just make the page apply to the up-to-date check and have it be something like "Up-to-date Checking." That's pretty specific and we're probably going to have more setttings needs come up in the future.

**Comment by davkean** [comment]
All the project systems implement up-to-date checks, why is ours special that it gets its own option?

**Comment by panopticoncentral** [comment]
The other project systems, I guess, haven't wanted to expose the ability to turn it on or off, or print logging. That's kind of a shame, since when up to date checks fail, it's hard to figure out what's going wrong without those options.

We've also gotten requests to expose controls for up to date checks in the old project system (#2412). So it's possible we'll extend the meaning of those options to the old project system too.

At this point, I'm inclined to change the name to "C#/VB/F# Project Settings" and have it be a general page that we can use to have options for the project system. Objections?

**Comment by SolderedMushroom** [comment]
@panopticoncentral If the name is too long, you could just call it "Project Settings" and put (presumably) all the options inside one of those box-outline thingies with a label of "C#, VB, F#".

This is not even unprecedented in VS options:

![image](https://user-images.githubusercontent.com/10489260/27242895-39c01a80-52ad-11e7-9280-f1c522c99eb0.png)

The whole tab is in one of those box-outline-label thingies.

**Comment by etbyrd** [comment]
Resolved in https://github.com/dotnet/project-system/pull/5071
